repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkIMain.scala

```diff
@@ -1129,7 +1129,7 @@ class SparkIMain(@BeanProperty val factory: ScriptEngineFactory, initialSettings
 
     def apply(line: String): Result = debugging(s"""parse("$line")""")  {
       var isIncomplete = false
-      currentRun.reporting.withIncompleteHandler((_, _) => isIncomplete = true) {
+      currentRun.parsing.withIncompleteHandler((_, _) => isIncomplete = true) {
```

Comment:
This was the change, it corresponds to https://github.com/scala/scala/commit/64ebac245d58221814f9c9375927e3f2e7a2d4f0